.github/workflows/night-ecr.yml

```diff
@@ -0,0 +1,44 @@
+name: Test ECR Publishing
+on:
+  # schedule:
+  #  - cron: '0 7 * * * ' ## Every day at 0700 UTC
+
+  workflow_dispatch: ## Give us the ability to run this manually
+
+
+# Abort prior jobs in the same workflow / PR
+concurrency:
+  group: ecr-publish-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  validate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Setup golang
+        uses: ./.github/actions/golang
+
+      - name: Setup NodeJS
+        uses: ./.github/actions/node
+
+      - name: Build the Zarf binary
+        run: make zarf-cli-linux-amd
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          role-to-assume: "zarf-nightly-tester"
+          aws-region: us-east-1
+
+      - name: Login to the ECR Registry
+        run: aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws/t8y5r5z5                                                                                                          ─╯
```

Comment:
public.ecr.aws/t8y5r5z5 this is kinda interesting - I guess that's a thing AWS does to obfuscate stuff